src/doc/gui_manual/Operators/OperatorTypes/PICS/index.rst

```diff
@@ -0,0 +1,268 @@
+Parallel Integral Curve System
+==============================
+
+Within the VisIt infrastructure is the ability to generate integral curves:
+streamlines and pathlines. An integral curve is a curve that begins at a seed
+location X, and is tangent at every point in a vector field. An integral curve
+is computed by numerical integration of the seed location through the vector
+field. The image below shows fieldlines through the magnetic field of a
+core-collapse supernova simulation from the GenASiS code.
+
+IMAGE HERE
+
+The generation of the curves form the basis of VisIt's Parallel Integral Curve
+System (PICS), made up of the :ref:`Integral Curve operator`, the :ref:`LCS
+operator`, the :ref:`Limit Cycle operator`, and the :ref:`Poincare operator`.
+Much of the underlying infrastructure and interface is the same for each
+operator. The user selects a series of seed locations where curves are
+generated, which are then visualized and analyzed.
+
+The PICS allows for the computation of Lagrangian Coherent Structures (LCS)
+using a variety techniques developed by George Haller and his group at ETH
+Zürich. Haller proposed that the ridges of Finite Time Lyapunov Exponents
+(FTLE) are indicators of hyperbolic LCS (repelling and attracting). In two
+dimensional flows, hyperbolic LCS's turn out to be stationary curves of the
+averaged material shear while elliptic LCS's are stationary curves of the
+averaged strain. The LCS operator coupled with the Integral Curve (Hyperbolic
+LCS) and Limit Cycle (Elliptical LCS) operators provide a mechanism to
+calculate these structures and have been developed in cooperation between the
+VisIt PICS team and the Haller Group.
+
+INCLUDE IMAGE of elliptical and hyperbolic LCS
+
+For complete details on computing elliptical LCS (closed orbits) and hyperbolic
+LCS (stretch and shrink lines) see the following paper: `K. Onu, F. Huhn, & G.
+Haller, LCS Tool: A Computational platform for Lagrangian coherent structures,
```

Comment:
Ok this looks like a PDF ref. I tested the current rendering of these docs and it works. That said, because this is a key ref, I might opt to actually copy the PDF into our docs so that we needen't worry about that link going stale. Not sure if that is appropriate though.

Reply:
I like the idea of guarding against the link going stale. Bad links in docs are always frustrating.

Reply:
Pdfs can be large in size (several megabytes). If we're going to put them in our repo, we should track them with git-lfs.

Reply:
> track them with git-lfs.

Just looked in .gitattributes. This is already done :)